### PR TITLE
Clarify trainee & Trainer uses for start points

### PR DESCRIPTION
**Pull request by karenword**
in response to @martinosorb feedback in Slack re a trainee who used this table to select an episode, rather than a lesson.

### CODE REVIEW

**Review by brownsarahm — APPROVED** [review]
I think we should merge this as is, but create an issue to address after the workbench transition that there be a version of this page that lists only the lessons, not the episodes for trainees.  It should be one page, for maintenance reasons, two pages will get out of sync, but the extra, episode-level information is mostly confusing. 

editing to note that the two-versions has to wait until after transition because workbench *enables* instructor and learner views of a page

**Comment by tobyhodges** [comment]
I wonder if it would be more effective to also adjust the column headers of the tables themselves? e.g.

This would have more impact on folks who scroll past the introductory text.

```markdown
|  Lesson   | (For Trainers) Good Starting Points | (For Trainers) Avoid | Notes     |
|---------- |------------------------------------ |--------------------- |---------- |
| [Data Organization in Spreadsheets](https://datacarpentry.org/spreadsheet-ecology-lesson/) |  any episode | none || 
```

**Comment by karenword** [comment]
Thanks all -- also tagging @klbarnes20 because this will fit in well with our checkout project later in Q3

**Comment by klbarnes20** [comment]
I agree with Sarah's comments and would support Toby's solution.

**Comment by ndporter** [comment]
A potential issue here (with @tobyhodges 's solution or otherwise) - there is critical information in some of the tables for trainees giving demos regardless. For example, both DC:Genomics and LC have lessons that are not suitable for demos. Currently that information is embedded only in the "Good Starting Points" and "Notes" field.

As an alternative, we could make the hyperlinks at the top into a nested list of the lessons we recommend (e.g. lessons as a second level under the curriculum titles). I'm guessing a nested list is also probably more screen-reader friendly than a table. Then we could add a main heading below the list like "For trainers: Suggested Starting Points" or even put them under a foldout if we want.

I agree they should be accessible to motivated trainees, but ultimately to avoid confusion we may need to put those charts on a different page if we want to emphasize the lessons only and not the episodes.

File: _extras/demo_lessons.md [hunk 1]
```diff
@@ -3,63 +3,102 @@ layout: page
...
+The list below shows lessons that have been reviewed for good and bad start points to use for a teaching demonstration. When selecting a lesson for a 
+teaching demonstration, trainees should review this list to ensure it is suitable.
 
-If the chosen episode is very short and finishes in less than five minutes, the trainee can continue on to the next episode in that lesson.
+Those who wish may also consult the detailed tables following the list to identify episodes marked "avoid" because they will not be asked to teach those episodes. However, we recommend 
+reviewing the full lesson regardless, as this will help provide context for other start points and is also good preparation for your eventual teaching 
+role!
+
+Note that while a trainee can choose the lesson to teach from, the **Trainer leading the demo session will choose the specific episode at the time
```
Comment (brownsarahm) [inline _extras/demo_lessons.md:13]:
```suggestion
Note that while a trainee can choose the lesson to teach from, the **Trainer** leading the demo session will choose the specific episode at the time 
```

File: _extras/demo_lessons.md [hunk 2]
```diff
@@ -3,63 +3,102 @@ layout: page
...
+The list below shows lessons that have been reviewed for good and bad start points to use for a teaching demonstration. When selecting a lesson for a 
+teaching demonstration, trainees should review this list to ensure it is suitable.
 
-If the chosen episode is very short and finishes in less than five minutes, the trainee can continue on to the next episode in that lesson.
+Those who wish may also consult the detailed tables following the list to identify episodes marked "avoid" because they will not be asked to teach those episodes. However, we recommend 
+reviewing the full lesson regardless, as this will help provide context for other start points and is also good preparation for your eventual teaching 
+role!
+
+Note that while a trainee can choose the lesson to teach from, the **Trainer** leading the demo session will choose the specific episode at the time
```
Comment (brownsarahm) [inline _extras/demo_lessons.md:13]:
```suggestion
Note that while a trainee can choose the lesson to teach from, the **Trainer leading the demo session will choose the specific episode at the time 
```

File: _extras/demo_lessons.md [hunk 3]
```diff
@@ -3,63 +3,102 @@ layout: page
...
+teaching demonstration, trainees should review this list to ensure it is suitable.
 
-If the chosen episode is very short and finishes in less than five minutes, the trainee can continue on to the next episode in that lesson.
+Those who wish may also consult the detailed tables following the list to identify episodes marked "avoid" because they will not be asked to teach those episodes. However, we recommend 
+reviewing the full lesson regardless, as this will help provide context for other start points and is also good preparation for your eventual teaching 
+role!
+
+Note that while a trainee can choose the lesson to teach from, the **Trainer leading the demo session will choose the specific episode at the time 
+of the demo session.** If you would like to use a lesson that is not listed here, you may contact the Instructor Trainer leading your teaching 
+demonstration or (if no one is listed for your session) email [checkout@carpentries.org](mailto:checkout@carpentries.org).
```
Comment (brownsarahm) [inline _extras/demo_lessons.md:15]: while this is open should this address be changed? @karenword

**Comment by brownsarahm** [comment]
I like the expanded list, and I think maybe we should put the list first and then the text about choosing an episide after the list.  That way we put the confusing part *after* the important part?

File: _extras/demo_lessons.md [hunk 4]
```diff
@@ -3,63 +3,102 @@ layout: page
...
 
+### <a id="dc-astro"></a> Data Carpentry: Astronomy
+
+| Lesson  | Good Starting Points | Avoid | Notes
+|---------- |---------- |---------- |---------- |---------- |---------- |
+| [Foundations of Astronomical Data Science](https://datacarpentry.org/astronomy-python/) | * [Basic Queries](https://datacarpentry.org/astronomy-python/01-query/index.html) <br/> * [Coordinate Transformations](https://datacarpentry.org/astronomy-python/02-coords/index.html) | any other episode |  Later episodes have dependencies. <br/> For [Basic Queries](https://datacarpentry.org/astronomy-python/01-query/index.html) skip the episode introduction, _Query Language_, and _Using Jupyter_ sections, and begin with [_Connecting to Gaia_](https://datacarpentry.org/astronomy-python/01-query/index.html#connecting-to-gaia) (`from astroquery.gaia import Gaia`)  |
 
 ### <a id="dc-astro"></a> Data Carpentry: Astronomy
```
Comment (tobyhodges) [inline _extras/demo_lessons.md:104]: Looks like the DC Astro info is now duplicated here? Also, the column headings need updating to become consistent with the changes being made to the other tables.

**Review by tobyhodges — CHANGES_REQUESTED** [review]

File: _extras/demo_lessons.md [hunk 5]
```diff
@@ -3,63 +3,102 @@ layout: page
...
-
-
 ### <a id="dc-genomics"></a>Data Carpentry: Genomics
 
 If you are an instructor in training and wish to use lessons from Data Carpentry’s Genomics curriculum for your teaching demo, please read [these instructions](https://datacarpentry.org/genomics-workshop/teaching_demos/index.html) to be sure you are prepared. You must follow these steps before your teaching demo, or you will be asked to reschedule.
 
-| Lesson  | Good Starting Points | Avoid | Notes
+|  Lesson   | (For Trainers) Good Starting Points | (For Trainers) Avoid | Notes     
 |---------- |---------- |---------- |---------- |---------- |---------- |
 | [Project Organization and Management for ](https://datacarpentry.org/organization-genomics/) | none | all lessons | Discussion based. No live coding. |
```
Comment (tobyhodges) [inline _extras/demo_lessons.md:75]:
```suggestion
| [Project Organization and Management for Genomics](https://datacarpentry.org/organization-genomics/) | none | all episodes | Discussion based. No live coding. |
```

A couple typos in the page that could be fixed in this PR.

File: _extras/demo_lessons.md [hunk 6]
```diff
@@ -3,63 +3,102 @@ layout: page
...
 | [Data Analysis and Visualization in R](https://datacarpentry.org/genomics-r-intro/) | none  | all episodes | This lesson is not yet stable. |
 
-
 ### <a id="dc-socialsci"></a>Data Carpentry: Social Sciences
 
-| Lesson  | Good Starting Points | Avoid | Notes
+|  Lesson   | (For Trainers) Good Starting Points | (For Trainers) Avoid | Notes     
 |---------- |---------- |---------- |---------- |---------- |---------- |
 | [Data Organization in Spreadsheets](https://datacarpentry.org/spreadsheets-socialsci/) | * [Dates as Data](https://datacarpentry.org/spreadsheets-socialsci/03-dates-as-data/index.html)<br/> * [Quality Assurance](https://datacarpentry.org/spreadsheets-socialsci/04-quality-assurance/index.html) | * [Introduction](https://datacarpentry.org/spreadsheets-socialsci/00-intro/index.html)<br/> * [Formatting Data Tables in Spreadsheets](https://datacarpentry.org/spreadsheets-socialsci/01-format-data/index.html)<br/> * [Formatting problems](https://datacarpentry.org/spreadsheets-socialsci/02-common-mistakes/index.html)<br/> * [Exporting data](https://datacarpentry.org/spreadsheets-socialsci/05-exporting-data/index.html) | Lessons listed to avoid have no live coding. |
```
[the remark this answers is not in the document]
Comment (brownsarahm) [reply _extras/demo_lessons.md:85]: great catch @tobyhodges !

File: _extras/demo_lessons.md (hunk 3 above)
Comment (brownsarahm) [inline _extras/demo_lessons.md:15]:
```suggestion
demonstration or (if no one is listed for your session) email [instructor.training@carpentries.org](mailto:checkout@carpentries.org).  
```

Comment (klbarnes20) [reply _extras/demo_lessons.md]: Yes, it should be changed to instructor.training@carpentries.org

File: _extras/demo_lessons.md [hunk 7]
```diff
@@ -3,58 +3,92 @@ layout: page
...
+teaching demonstration, trainees should review this list to ensure it is suitable.
 
-If the chosen episode is very short and finishes in less than five minutes, the trainee can continue on to the next episode in that lesson.
+Those who wish may also consult the detailed tables following the list to identify episodes marked "avoid" because they will not be asked to teach those episodes. However, we recommend 
+reviewing the full lesson regardless, as this will help provide context for other start points and is also good preparation for your eventual teaching 
+role!
+
+Note that while a trainee can choose the lesson to teach from, the **Trainer leading the demo session will choose the specific episode at the time 
+of the demo session.** If you would like to use a lesson that is not listed here, you may contact the Instructor Trainer leading your teaching 
+demonstration or (if no one is listed for your session) email [instructor.training@carpentries.org](mailto:checkout@carpentries.org).
```
Comment (brownsarahm) [inline _extras/demo_lessons.md:15]:
```suggestion
demonstration or (if no one is listed for your session) email [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).  
```

**Comment by brownsarahm** [comment]
I think this is worth merging, the checkout project may change things later, but I think this text is likely helpful

**Pull request merged**